app/controllers/glossary_controller.rb

```diff
@@ -0,0 +1,4 @@
+class GlossaryController < ApplicationController
+  def show
```

Comment:
true, but I find it easier to follow the code when an action does exist - just preference.